app/graphql/types/test_result.rb

```diff
@@ -3,6 +3,8 @@
 module Types
   # Definition of the Profile type in GraphQL
   class TestResult < Types::BaseObject
+    (enforce_rbac Rbac::REPORT_READ) || (enforce_rbac Rbac::REPORT_EXPORT_READ)
```

Comment:
I don't think this would work, if the first one returns anything non-nil, it will never run the second one. If you run the second one, it'll overwrite the `@rbac_permission`. I think you want something like:
```suggestion
    enforce_rbac Rbac::REPORT_READ, Rbac::REPORT_EXPORT_READ
```

And please write tests for it.